src/redux/Books/Books.js

```diff
@@ -0,0 +1,47 @@
+const ADD_BOOK = 'addBook';
+const REMOVE_BOOK = 'removeBook';
```

Comment:
- [ ] It 's a good thing you implemented the changes requested by the previous code reviewer, but as a requirement for this project and best practice, please we are supposed to use the [ducks pattern](https://github.com/erikras/ducks-modular-redux) to define our action types like so(just an example); `const ADD_BOOK = bookstore/books/ADD_BOOK`, `const REMOVE_BOOK = bookstore/books/REMOVE_BOOK`

Reply:
Thank you for your review. I have solved this issue.